stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ImageUtils.kt

```diff
@@ -153,6 +160,7 @@ internal fun StreamAsyncImage(
             val context = LocalContext.current
             val imageAssetTransformer = ChatTheme.streamImageAssetTransformer
             val imageHeaderProvider = ChatTheme.streamImageHeadersProvider
+            val fetchRetries = remember { mutableIntStateOf(0) }
```

Comment:
nit: we could remove the need to use `intValue`  if using delegation when declaring `fetchRetries`:

```kotlin
var fetchRetries by remember { mutableIntStateOf(0) }
```

That way, you could just do `if (fetchRetries++ < MaxRetries) {`